### PR TITLE
Rename the folder of rhel8 workflow

### PR DESCRIPTION
**Pull request by jinyunma**
1. Rename the folder for rhel8 workflow.
workers-rhel8: only have rhel8 workers, rhcos wokers are removed
workers-rhcos-rhel8: have both rhcos workers and rhel8 workers
2. Introduce new env var `RHEL_VM_DISK_SIZE` to set disk size on azure/vsphere provision ref
3. Extend lv of /var for RHEL machine on Azure platform

### CODE REVIEW

**Comment by liangxia** [comment]
There are several destructive changes, e.g, workflow rename, var rename.
I would suggest break them into several smaller PRs, each one contains only one destructive changes.

**Comment by jinyunma** [comment]
@liangxia thanks for the suggestion, since all changes are under azure/vsphere rhel8 chain except workflow rename, so I updated in one PR, and only two jobs are affected.

File: ci-operator/step-registry/workers-rhel/pre-hook/azure/workers-rhel-pre-hook-azure-commands.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+cat > scaleup-pre-hook-azure.yaml <<- 'EOF'
+- name: configuraiton RHEL machine on Azure
+  hosts: new_workers
+  any_errors_fatal: true
+  gather_facts: false
+  tasks:
+  - name: Extend the lv of /var for RHEL machine
+    shell: |
+      sudo lvextend -L+20G /dev/rootvg/varlv
```
Comment (jianlinliu) [inline ci-operator/step-registry/workers-rhel/pre-hook/azure/workers-rhel-pre-hook-azure-commands.sh:27]: Let us replace `lvextend -L+20G` with `lvextend -l +100%FREE` to make it a bit wiser.

Comment (jianlinliu) [reply ci-operator/step-registry/workers-rhel/pre-hook/azure/workers-rhel-pre-hook-azure-commands.sh]: That means there are some free disk spaces not allocated yet, if we want to use the full size of disk, need to create a new partition, use it to create a pv, then add the pv to vg, then lvextend again. Per our QE past testing in jenins ci pipleine, seem like 49G /var is enough for testing. So we do not need to set 128G disk for now, setting it to 80G by default for `RHEL_VM_DISK_SIZE` should be enough in workers-rhel-azure-provision-ref.yaml, then drop other places `RHEL_VM_DISK_SIZE` setting.

Comment (jinyunma) [reply ci-operator/step-registry/workers-rhel/pre-hook/azure/workers-rhel-pre-hook-azure-commands.sh]: agree, I saw that size of disk /var is 20G on vsphere rhel worker, and seems not hit disk pressure issue so far. I updated to set 64G by default in provision ref, which is same value as in `v4-sacleup` repo.

**Comment by jinyunma** [comment]
@jianlinliu @liangxia please help to review the update.
The job release-4.11-amd64-nightly-e2e-vsphere-ipi-proxy-workers-rhcos-rhel8 failed twice.
Hit the [Bug 2089153](https://bugzilla.redhat.com/show_bug.cgi?id=2089153) on the first time. 
It's strange on the second time, there is no any logs collected for control plane nodes in bootstrap log bundle. But from the screenshot of remote console for control plane nodes, node booted up and os was updated to version embedded in payload, difficult to identify the root cause, will check through Jenkins job later.
Anyway, the failure should unrelated with this PR, since installer rehearse job is passed.

**Pull request merged**